frontend/public/components/nav/admin-nav.tsx

```diff
@@ -326,7 +326,7 @@ const AdminNav = () => {
         <ResourceNSLink
           id="rolebindings"
           resource="rolebindings"
-          name={t('nav~Role Bindings')}
+          name={t('nav~RoleBindings')}
```

Comment:
I found under one use/group page, the tab "Role Bindings" are not shown as "RoleBindings", need they be updated here?

Reply:
Good catch! Thanks @yanpzhan.

Reply:
Addressed this (and a couple more I found on a second pass). Thanks again!